### ISSUE TITLE
show acronym when autosuggesting preferred terms

### ISSUE THREAD

**Issue by hancush** [datamade/la-metro-councilmatic#453]
e.g., "preferred term (acronym: **matched**)"

**Comment by hancush** [comment]
n.b., the `values` object for each hint includes whether the query is the preferred term (`PT`) or the non-preferred term (`NPT`, a.k.a. acronym):

```
"termHints" : [ {
    "name" : "Preferred Label Here",
    "id" : "concept UUID (Semaphore)",
    "classes" : [ "Concept Class Here" ],
    "values" : [ {
      "pre_em" : "label text before the match",
      *** "nature" : "PT|NPT", ***
      "em" : "matched text here",
      "post_em" : "label text after the match",
      "value" : "matched concept label here (could be preferred label or alt label"
    } ],
    "facets" : [ {
      "name" : "concept scheme name here",
      "id" : "concept scheme UUID"
    } ]
```

for example, if you query `lrtp`, it will suggest `long range transportation plan` with a `nature` value of `NPT`.

**Comment by jmithani** [comment]
so in the example above, the suggestion would be "Long Range Transportation Plan **(LRTP)**" then?

**Comment by hancush** [comment]
@jmithani yes

**Comment by hancush** [comment]
closed in #445!
